fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -88,20 +90,23 @@ class GridSearch(ReductionsEstimator):
     _MESSAGE_X_SENSITIVE_ROWS = "X and the sensitive features must have same number of rows"
 
     def __init__(self,
-                 learner,
+                 estimator,
                  disparity_metric,
-                 quality_metric,
+                 selection_rule=_TRADEOFF_OPTIMIZATION,
+                 constraint_weight=0.5,
```

Comment:
Are there limits on the range of sensible values for `constraint_weight` ? I'm guessing [0, 1] ?

Reply:
In fact, reading more of the code below, I'm pretty certain that that's the required range.